frontend/views/pages/PayGroup.vue

```diff
@@ -1,122 +1,128 @@
 <template lang="pug">
-page(pageTestName='payGroupPage' pageTestHeaderName='payGroupTitle')
+page(
+  pageTestName='payGroupPage'
+  pageTestHeaderName='payGroupTitle'
+)
   template(#title='') Pay Group
 
-  template(#sidebar='')
-    h2.subtitle
-      i18n Payments Sent
-
-    h3
-      i18n(
-        :args='{ sent: paymentSummary.sent, total: fakeStore.users.length }'
-      ) {sent} of {total}&nbsp;
-
-      tooltip(v-if='paymentSummary.hasWarning')
-        i.icon-exclamation-triangle
-
-        template(slot='tooltip')
-          strong Payment Declined
-          i18n(tag='p')
-            | Someone didn&rsquo;t confirm your payment. Please mark as payed only when it&rsquo;s done.
-
-    h2.subtitle
-      i18n Payments Confirmed
-
-    h3
-      i18n(:args='{ sent: paymentSummary.confirmed, total: fakeStore.users.length }')
-        | {sent} of {total}
-
-    h2.subtitle
-      i18n Amount Sent
-
-    h3(:class="{'has-text-success': paymentAllDone}")
-      i18n(
-        v-if='paymentAllDone'
-        :args='{ currency: fakeStore.currency, amountTotal: paymentSummary.amountTotal }'
-      ) All {currency}{amountTotal}
-
-      i18n(
-        v-else=''
-        :args='{ currency: fakeStore.currency, amoutPayed:paymentSummary.amoutPayed, amountTotal: paymentSummary.amountTotal }'
-      ) {currency}{amoutPayed} of {currency}{amountTotal}
-
-      progress-bar(
-        :primary='paymentProgress.sent'
-        :secondary='paymentProgress.confirmed'
+  template(#sidebar=''
+    v-if='hasPayments'
+  )
+    .c-summary-item(
+      v-for='(item, index) in paymentSummary'
+      :key='index'
+    )
+      i18n.title.is-4(tag="h4") {{item.title}}
```

Comment:
`i18n` tags never take variables, they always take literals. So this needs to be:

```js
h4.title.is-4 {{ item.title }}
```

Reply:
fixed